### PR TITLE
Copy-Edit-How-To-Ask-For-Help-With-Your-Code

### LINKED ISSUE #347: https://github.com/Umuzi-org/ACN-syllabus/tree/develop/content/topics/how-to-ask-for-help-with-your-code/_index.md

### PR DESCRIPTION
**Pull request by asradebe**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [ ] I ran the hugo server and looked at my changed in the browser with my own eyes
- [ ] I ran the linter and there were no errors
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: content/topics/how-to-ask-for-help-with-your-code/_index.md [hunk 1]
```diff
@@ -33,22 +33,22 @@ staff member: Here is an explanation [explanation], and here are some resources
 #### Where to ask good questions
 
 - [Stackoverflow - How to ask for help](https://stackoverflow.com/help/how-to-ask) == TRUELY **AMAZING**
-- Any tech community discussion that you are a part of
+- Any tech community discussion that you are a part of(Discord, RocketChat)
 
 #### Characteristics of a good question:
 
-As a general rule: If you really want help then make it as easy as possible for people to help you. Empathise with the person who will help you. If they are left guessing or they have to ask a million follow up questions then that means you asked a bad-quality question.
+As a general rule: If you want help then make it as easy as possible for people to help you. Empathize with the person who will help you. If they are left guessing or they have to ask a million follow-up questions then that means you asked a bad-quality question.
```
Comment (JacobUmuzi) [inline content/topics/how-to-ask-for-help-with-your-code/_index.md:40]: change Empathize to British English

Comment (JacobUmuzi) [inline content/topics/how-to-ask-for-help-with-your-code/_index.md:40]: change Empathize to British English

File: content/topics/how-to-ask-for-help-with-your-code/_index.md [hunk 2]
```diff
@@ -33,22 +33,22 @@ staff member: Here is an explanation [explanation], and here are some resources
 #### Where to ask good questions
 
 - [Stackoverflow - How to ask for help](https://stackoverflow.com/help/how-to-ask) == TRUELY **AMAZING**
-- Any tech community discussion that you are a part of
+- Any tech community discussion that you are a part of(Discord, RocketChat)
```
Comment (JacobUmuzi) [inline content/topics/how-to-ask-for-help-with-your-code/_index.md]: put a space before parentheses

Comment (JacobUmuzi) [inline content/topics/how-to-ask-for-help-with-your-code/_index.md]: put a space before parentheses

File: content/topics/how-to-ask-for-help-with-your-code/_index.md [hunk 3]
```diff
@@ -33,22 +33,22 @@ staff member: Here is an explanation [explanation], and here are some resources
...
-- A similar good way to use [Google](https://www.google.com) is when searching for something specific, try using quotes to minimize the guesswork for Google search. When you put your search parameters in quotes, it tells the search engine to search for the whole phrase.
\ No newline at end of file
+- In general, if your question text contains code or error message those things need to be formatted properly. It needs to not look like a jumbled mess of words and symbols. Different platforms have different ways of pulling this off. RocketChat uses the backtick symbol. And Stackoverflow has its way. Be aware of the tools and conventions of the platform you are using, then follow those conventions.
+- A similar good way to use [Google](https://www.google.com) is when searching for something specific, try using quotes to minimize the guesswork for Google search. When you put your search parameters in quotes, it tells the search engine to search for the whole phrase.
+
+**NB.** IT IS YOUR RESPONSIBILITY TO HELP OTHERS WHEN AND IF YOU HAVE THE ANSWERS (Basically help and be helped).
```
Comment (JacobUmuzi) [inline content/topics/how-to-ask-for-help-with-your-code/_index.md:54]: Miniscule, first letter in the parentheses unless it is a noun or beginning of a complete grammatical sentence.

Comment (JacobUmuzi) [inline content/topics/how-to-ask-for-help-with-your-code/_index.md:54]: Miniscule, first letter in the parentheses unless it is a noun or beginning of a complete grammatical sentence.

File: content/topics/how-to-ask-for-help-with-your-code/_index.md [hunk 4]
```diff
@@ -33,22 +33,22 @@ staff member: Here is an explanation [explanation], and here are some resources
...
-- Any tech community discussion that you are a part of
+- Any tech community discussion that you are a part of(Discord, RocketChat)
 
 #### Characteristics of a good question:
 
-As a general rule: If you really want help then make it as easy as possible for people to help you. Empathise with the person who will help you. If they are left guessing or they have to ask a million follow up questions then that means you asked a bad-quality question.
+As a general rule: If you want help then make it as easy as possible for people to help you. Empathize with the person who will help you. If they are left guessing or they have to ask a million follow-up questions then that means you asked a bad-quality question.
 
-- Ask on a bigger audience platform. (e.g Asking on the #tech-assist channel helps you get to a much bigger audience and possibly quick responses from your fellow students, managers and alumni)
+- Ask on a bigger audience platform. (e.g Asking on the #General channel helps you get to a much bigger audience and possibly quick responses from your fellow students, managers, and alumni)
```
Comment (JacobUmuzi) [inline content/topics/how-to-ask-for-help-with-your-code/_index.md]: maybe include that the general channel is on rocket chat

Comment (JacobUmuzi) [inline content/topics/how-to-ask-for-help-with-your-code/_index.md]: maybe include that the general channel is on rocket chat

**Review by JacobUmuzi — CHANGES_REQUESTED** [review]

File: content/topics/how-to-ask-for-help-with-your-code/_index.md (hunk 3 above)
Comment (asradebe) [reply content/topics/how-to-ask-for-help-with-your-code/_index.md:54]: what is miniscule? and what line?

File: content/topics/how-to-ask-for-help-with-your-code/_index.md (hunk 1 above)
Comment (asradebe) [reply content/topics/how-to-ask-for-help-with-your-code/_index.md:40]: Grammarly doesn't suggest a change. it recognises the word empathize in British English so I'm not sure lol

File: content/topics/how-to-ask-for-help-with-your-code/_index.md (hunk 3 above)
Comment (asradebe) [reply content/topics/how-to-ask-for-help-with-your-code/_index.md:54]: there's no grammar error here from my side and it's in British English.

File: content/topics/how-to-ask-for-help-with-your-code/_index.md (hunk 4 above)
Comment (asradebe) [reply content/topics/how-to-ask-for-help-with-your-code/_index.md]: Done

File: content/topics/how-to-ask-for-help-with-your-code/_index.md (hunk 2 above)
Comment (asradebe) [reply content/topics/how-to-ask-for-help-with-your-code/_index.md]: done

**Review by JacobUmuzi — APPROVED** [review]

**Pull request merged**